### PR TITLE
Update library naming

### PR DESCRIPTION
**Pull request by JonnavithulaGirish**
## Related Issues
- Add LICENSE.
- Change library naming.

### Internal Changelog
- Add license file - default MIT template
- Update `WebView` library naming to `webview` 
- New pakage would look as `io.transcend.webview:webview:1.0.0`

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -15,4 +15,4 @@ dependencyResolutionManagement {
 
 rootProject.name = "ConsentWebView"
 include ':app'
-include ':WebView'
+include ':webview'
```
Comment (MadDataScience) [inline settings.gradle:18]: Curious why this change?

**Review by MadDataScience — APPROVED** [review]

File: settings.gradle (hunk 1 above)
Comment (JonnavithulaGirish) [reply settings.gradle:18]: Just following naming schema that was missed earlier.

Generally most of the libraries are expose as 

```
'com.google.android.material:material:1.9.0'
'androidx.constraintlayout:constraintlayout:2.1.4'
```
which is
`{groupId}.{projectId}:{projectId}:{version}`

Our previous implantation wasn't following this, which is why we were using
`
io.transcend.webview:WebView:1.0.0-SNAPSHOT `

Instead of

`io.transcend.webview:webview:1.0.0-SNAPSHOT `

Comment (MadDataScience) [reply settings.gradle:18]: Interesting. For my own edification, is this simply a matter of conforming to convention? Or was something actually broken?

Comment (JonnavithulaGirish) [reply settings.gradle:18]: Just a matter of convention. Even without this change our library would work but just to avoid  naming confusion for clients renamed our project.
